coreos-base/oem-ec2-compat/files/grub.cfg

```diff
@@ -0,0 +1,4 @@
+# CoreOS GRUB settings for EC2
```

Comment:
Can you give this file an ec2-specific name? (in case we get other grub configs in the future)